action.yml

```diff
@@ -18,12 +18,18 @@ inputs:
   swarm-id:
     required: false
     description: 'ID of the swarm. Only required if you deploy to a swarm.'
+  endpoint-id:
+    required: true
```

Comment:
Can we make this optional instead and then default to 1 to ensure backward compatible?